landbosse/model/CollectionCost.py

```diff
@@ -378,12 +379,20 @@ def calc_num_turb_partial_strings(num_leftover_turb, num_turb_per_cable):
             if num_remaining > 0:
                 turb_per_partial_string.append(min(num_remaining, max_turb))
             else:
+                # Note: This will create a nan in the division below, but it is
+                # detected and fixed if that happens.
+                print(f'Warning {self.project_name} has {num_turbines} turbines at {turbine_rating_MW} each. While calculating partial strings, found num_remaing to be {num_remaining}. This will cause divsion by 0 which will be corrected.')
                 turb_per_partial_string.append(0)
```

Comment:
It doesn't look like this list is the one that's used in the denominator so it shouldn't cause any problems if these values are zero. I've made some changes to address this issue.